### PR TITLE
KAZOO-5912: Fixed leg creation when undefined privacy prefs

### PR DESCRIPTION
**Pull request by sergey-safarov**
For more details please looks [KAZOO-5912](https://2600hz.atlassian.net/browse/KAZOO-5912)

### CODE REVIEW

File: applications/callflow/src/module/cf_resources.erl [hunk 1]
```diff
@@ -372,7 +372,7 @@ handle_channel_destroy(_, JObj) ->
 get_privacy_prefs(Call) ->
     case use_endpoint_prefs(Call) of
         'true' -> check_inception(Call);
-        'false'  -> []
+        _  -> []
```
Comment (jamesaimonetti) [inline applications/callflow/src/module/cf_resources.erl]: I'd rather change `update_endpoint_prefs` and do `kapps_call:kvs_fetch(<<"use_endpoint_privacy">>, Call, 'false').` instead, and keep this case clause using `false`.

Comment (jamesaimonetti) [reply applications/callflow/src/module/cf_resources.erl]: Err, `(<<"use_endpoint_privacy">>, 'false', Call)`.

**Comment by sergey-safarov** [comment]
Hello @jamesaimonetti 
PR is updated.

**Comment by jamesaimonetti** [comment]
And merged! Thanks :)

**Pull request closed without merging**